Composer/plugins/sample-ui-plugin/package.json

```diff
@@ -9,6 +9,7 @@
     "clean": "rimraf dist"
   },
   "composer": {
+    "enabled": false,
```

Comment:
can you change [the following line in the sample's README](https://github.com/microsoft/BotFramework-Composer/blame/main/Composer/plugins/sample-ui-plugin/README.md#L217) that talks about toggling the `extendsComposer` property to reflect this `composer.enabled` property

Reply:
Yes. Eventually, I think we move away from `extendsComposer`.